tests/file_handler/test_data_content_provider.py

```diff
@@ -102,17 +102,18 @@ def test_scan_zipfile_p(self) -> None:
                             with open(os.path.join(dirpath, filename), "rb") as input_file:
                                 output_file.write(input_file.read())
                                 samples_file_count += 1
-            assert samples_file_count == SAMPLES_FILES_COUNT
+            # self.assertEqual(SAMPLES_FILES_COUNT, samples_file_count)
```

Comment:
Could you please check this annotated line?

Reply:
done